Doc/c-api/list.rst

```diff
@@ -56,13 +56,26 @@ List Objects
    Similar to :c:func:`PyList_Size`, but without error checking.
 
 
+.. c:function:: PyObject* PyList_GetItemRef(PyObject *list, Py_ssize_t index)
+
+   Return the object at position *index* in the list pointed to by *list*.  The
+   position must be non-negative; indexing from the end of the list is not
+   supported.  If *index* is out of bounds (<0 or >=len(list)),
+   return ``NULL`` and set an :exc:`IndexError` exception.
+
+   .. versionadded:: 3.13
+
+
 .. c:function:: PyObject* PyList_GetItem(PyObject *list, Py_ssize_t index)
 
    Return the object at position *index* in the list pointed to by *list*.  The
    position must be non-negative; indexing from the end of the list is not
    supported.  If *index* is out of bounds (<0 or >=len(list)),
    return ``NULL`` and set an :exc:`IndexError` exception.
 
+   This behaves like :c:func:`PyList_GetItemRef`, but returns a
+   :term:`borrowed reference` instead of a :term:`strong reference`.
```

Comment:
Only line 76 and 77 is needed:

```suggestion
   Like :c:func:`PyList_GetItemRef`, but returns a
   :term:`borrowed reference` instead of a :term:`strong reference`.
```